src/Finder/TestFrameworkFinder.php

```diff
@@ -136,4 +140,19 @@ private function findTestFramework(): string
 
         throw FinderException::testFrameworkNotFound($this->testFrameworkName);
     }
+
+    private function findFromBatchFile(string $path): string
+    {
+        // Check for Composer generated or project proxy code
+        if (preg_match('/%~dp0(.+$)/mi', (string) file_get_contents($path), $match)) {
```

Comment:
It isn't immediately clear what is going checked for here, especially if one hasn't used anything Windows for some time. But this explanation would suffice:

> (~dp0 is an alias to $0 in bash world)

Reply:
Okay, I'll add that